<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>63</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>63</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>63</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>51</red>
        <green>63</green>
        <blue>80</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>人物模拟器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/image.qrc">
    <normaloff>:/icon/images/icon.png</normaloff>:/icon/images/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>741</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KaiTi</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 74, 94);
color: rgb(255, 255, 255);
border: 1px solid;
border-radius: 3px;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>体检报告单</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>7</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="key_1">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_1">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>工人</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_2">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>姓别：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_2">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>男</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_3">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>专业：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_3">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>信息科学技术</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="key_4">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>身高：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_4">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>180cm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_5">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>体重：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_5">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>74kg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_6">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>资产：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_6">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>3000元</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="key_7">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>体质：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_7">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>88</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_8">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>智力：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_8">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>76</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_9">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>魅力：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_9">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>78</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_10">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>能力：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_10">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>56</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="key_11">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>学分：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_11">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="key_12">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(114, 97, 127);</string>
      </property>
      <property name="text">
       <string>爱好：</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="value_12">
      <property name="font">
       <font>
        <family>HY그래픽M</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>滑滑板</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_refresh">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KaiTi</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(60, 74, 94);
border: 1px solid;
border-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>再体检一次</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>542</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QLabel" name="label_topic">
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>你拿到了入学时的体检报告：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="logo">
      <property name="maximumSize">
       <size>
        <width>67</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource/image.qrc">:/icon/images/logo_BIT.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>500</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btn_next">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 74, 94);
border: 1px solid;
border-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_prev">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>KaiTi</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(60, 74, 94);
border: 1px solid;
border-color: rgb(255, 255, 255);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
  <zorder>btn_refresh</zorder>
 </widget>
 <resources>
  <include location="resource/image.qrc"/>
 </resources>
 <connections/>
</ui>
